engine/execution/computation/computer/computer.go

```diff
@@ -449,39 +441,33 @@ func (e *blockComputer) executeTransaction(
 
 	memAllocAfter := debug.GetHeapAllocsBytes()
 
-	lg := e.log.With().
-		Str("tx_id", txID.String()).
-		Str("block_id", blockIdStr).
-		Str("traceID", traceID).
+	logger = logger.With().
 		Uint64("computation_used", tx.ComputationUsed).
 		Uint64("memory_used", tx.MemoryEstimate).
-		Uint64("memAlloc", memAllocAfter-memAllocBefore).
-		Int64("timeSpentInMS", time.Since(startedAt).Milliseconds()).
+		Uint64("mem_alloc", memAllocAfter-memAllocBefore).
+		Int64("time_spent_in_ms", time.Since(startedAt).Milliseconds()).
 		Logger()
 
 	if tx.Err != nil {
-		errMsg := tx.Err.Error()
-
-		lg.Info().
-			Str("error_message", errMsg).
+		logger = logger.With().
+			Str("error_message", tx.Err.Error()).
 			Uint16("error_code", uint16(tx.Err.Code())).
-			Msg("transaction execution failed")
+			Logger()
+		logger.Info().Msg("transaction execution failed")
 
 		if isSystemTransaction {
 			// This log is used as the data source for an alert on grafana.
 			// The system_chunk_error field must not be changed without adding
 			// the corresponding changes in grafana.
 			// https://github.com/dapperlabs/flow-internal/issues/1546
-			e.log.Error().
-				Str("error_message", errMsg).
```

Comment:
I wonder why we didnt just use `.Err(tx.Err)` here

Reply:
i have no idea, but the comment looks scary so i'm leaving this as is